.github/workflows/python_unit_test.yaml

```diff
@@ -28,22 +39,28 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-    
+
     steps:
       - name: Check out code
         uses: actions/checkout@v3
       - name: Set up python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.6.7"
+          python-version: ${{ inputs.python-version }}
       - name: Install poetry
         run: |
-            pip --version
-            pip install poetry
-            poetry install -vv
+          pip --version
+          pip install poetry
+          poetry install -vv
       - name: Service specific setup
         run: |
-            curl -sSL "${{inputs.setup-script-location}}" | bash
+          curl -sSL "${{inputs.setup-script-location}}" | bash
       - name: Run test
-        run: | 
-          poetry run pytest -vv --cov=fence --cov=migrations/versions --cov-report xml tests 
+        run: |
+          poetry run pytest -vv --cov=${{ env.REPO_NAME }} --cov=migrations/versions --cov-report xml tests
```

Comment:
`cov=migrations/versions` is also specific to fence. We need to be able to customize the pytest command. Can we make this a parameter that default to just `pytest -vv .`?